src/IIIFPresentation/API/Features/Storage/Helpers/ThumbnailX.cs

```diff
@@ -0,0 +1,33 @@
+﻿using IIIF.Presentation.V3.Content;
+
+namespace API.Features.Storage.Helpers;
+
+public static class ThumbnailX
+{
+    private const int ThumbnailSize = 100;
+    private const int AcceptableDelta = 2;
```

Comment:
Does this need to be restricted to a set size, or can it looks for size closest to? (which is how it's done in Protagonist https://github.com/dlcs/protagonist/blob/main/src/protagonist/DLCS.Model/IIIF/IIIFX.cs#L38)

Is it better to find _a_ thumb, rather than the perfect thumb?

Reply:
swapped to use `ClosestTo`